mod/gc_elgg_sitemap/start.php

```diff
@@ -67,7 +69,33 @@ function gc_elgg_sitemap_init() {
 		elgg_register_plugin_hook_handler('entity:url', 'object', 'redirect_content_url', 1);
 		elgg_register_plugin_hook_handler('view', 'output/longtext', 'strip_content_hyperlinks_handler');
 
+		// modifying view in the user's profile
+		elgg_register_plugin_hook_handler('view', 'b_extended_profile/portfolio', 'linkedin_profile_handler');
+
+	}
+}
+
+function linkedin_profile_handler($hook, $type, $return, $params) {
+
+	$user = elgg_get_page_owner_entity();
+	$portfolio_guid = $user->portfolio;
+
+	// when no portfolio for user exists
+	if (empty($portfolio_guid) || $portfolio_guid == NULL)
+		return;
+
+	if (!is_array($portfolio_guid))
+		$portfolio_guid = array($portfolio_guid);
+
+	foreach ($portfolio_guid as $guid) {
+		
+		$entry = get_entity($guid);
+		if ($entry instanceof ElggEntity) {
+			echo gc_explode_translation($entry->title, 'en').' / '.gc_explode_translation($entry->title, 'fr');
```

Comment:
an ElggEntity wouldn't necessarily have a title attribute, but in this context it should be fine.